### DISCUSSION TITLE
Can you add Quarto extensions created in an internal Github repo (from within a Github Organization)?

### DISCUSSION THREAD

**Discussion by parmsam-pfizer** [quarto-dev/quarto-cli#10145] [Feature Requests]
### Description

Quarto extensions shared in public Github repos seem to work for me fine. However, I get the following error when trying to add a Quarto extension (for a custom RevealJS theme) that I've created within an internal repo within a private Github Org: 
```
$ quarto use template github-org-name-here/quarto-custom-revealjs
Extension not found in local or remote sources
```

Not sure if this is related to https://github.com/quarto-dev/quarto-cli/issues/5827. Should I create a separate issue for this since it applies to repos within private Github Orgs? Have others had success sharing (and adding via Quarto CLI) Quarto extensions within private Github orgs?

**Comment by cscheid** [comment]
That syntax requires a public repository. With that said, you can always clone the extension and refer the local path directly.

**Reply by parmsam-pfizer** [reply]
Are private repos or repos within private Github Orgs on the roadmap?

**Reply by cscheid** [reply]
Not currently, no.

**Reply by parmsam-pfizer** [reply]
Updated category to feature request. This would be helpful for Quarto extensions that are created within orgs and cannot be shared publicly/externally.

**Reply by mcanouil** [reply]
Extensions can be installed from the assets as well.
Download the asset, install it: https://quarto.org/docs/extensions/distributing.html

**Reply by mcanouil** [reply]
I am closing this as it is a duplicate:

- https://github.com/quarto-dev/quarto-cli/discussions/4444

Which was closed by:

- https://github.com/quarto-dev/quarto-cli/issues/5827
